autosklearn/__init__.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 from autosklearn.util import dependencies
-from autosklearn.__version__ import __version__
+from autosklearn.__version__ import __version__  # noqa (imported but unused)
```

Comment:
Is this import really necessary?

Reply:
Yes, it allows `print(autosklearn.__version__)`

Reply:
ok